client/public/elements/views/map/app-map-info-panel.js

```diff
@@ -164,35 +252,68 @@ export default class AppMapInfoPanel extends Mixin(LitElement)
       // find connections
       let connections = [];
       let link;
+      let check;
       for( let id in this.graph.links ) {
         link = this.graph.links[id];
-        if( link.src === node.id ) {
-          connections.push({
-            link,
-            node : this.graph.nodes[link.dst]
-          });
-        } else if ( link.dst === node.id ) {
+        if ( link['@id'].includes(node['@id']) ) {
```

Comment:
@ldragoon why is the '@id' property an array here?

Reply:
I'm confused on this one, would appreciate advice.